Documentation air-gap clarification (#861)
k0sproject/k0s: docs/airgap-install.md

#### 1. Create OCI bundle

> To create an OCI bundle, use the same computer architecture (x86-64, ARM64, ARMv7) as you'll have in the target system, because the OCI bundles are build specifically for certain architecture.

Reviewer: ```suggestion
To create an OCI bundle, use the same processor architecture (x86-64, ARM64, ARMv7) as on the target system, because the OCI bundles are built specifically for each architecture.
```

Author: Great comments!! Updated.
